src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
```

Comment:
If you check the System.out.println(obj) implementation you will see that the method is using the object you passed to get its string value by calling "obj.toString()" so you don't need to call "toString()" the second time.